dev-docs/plugins/cross-player-prebid-component/cross-player-config.md

```diff
@@ -0,0 +1,411 @@
+---
+layout: page_v2
+title: Cross-Player Prebid Component Configuration
+description: Configuring the cross-player Prebid component
+top_nav_section: dev_docs
+nav_section: plugins
+---
+
+# Cross Player Prebid Component Configuration
+{:.no_toc}
+
+Options that configure the behavior of the Prebid process are passed to the [Cross-Player Prebid Component]({{site.baseurl}}/dev-docs/plugins/cross-player-prebid-component/about-cross-player-prebid-component.html) in a JSON structure.  These options include the bidders and bidder settings for Prebid.js to use, the URL to your custom Prebid.js build, and any ad server parameters.
+
+The Prebid Component supports almost all the Prebid options supported by the other Prebid plugins, with some exceptions.  The following is the list of Prebid options supported in the Component:
+
+- [prebidPath](#prebidPath)
+- [biddersSpec](#biddersSpec)
+- [bidderAliases](#bidderAliases)
+- [bidderSettings](#bidderSettings)
+- [prebidConfigOptions](#prebidConfigOptions)
+- [dfpParameters](#dfpParameters)
+- [prebidTimeout](#prebidTimeout)
+- [enablePrebidCache](#enablePrebidCache)
+
+<a name="prebidPath"></a>
+## prebidPath
+
+**Description:**
+
+The path to Prebid.js that you wish to use.
+
+NOTE: By default, the plugin currently supports Prebid.js 1.0 and up. It does NOT support version 0.x due to the breaking changes that were added in Prebid.js 1.0
```

Comment:
Suggest dropping the breaking change part. "It does NOT support version 0.x." is enough. 0.x is several years old.

Reply:
I have removed that phrase from the internal doc.  It can be removed from the doc present in this pull request.

Reply:
Same Q -- @jeanstemp ?

Reply:
I believe this DOC project has been assigned to @SSuranie  - so Steven - are you going to make the suggested changes to this pull request?

Reply:
The breaking change text has been removed.